src/install/kafka/whatsNext.mdx

```diff
@@ -0,0 +1,4224 @@
+        Required to produce KafkaProducerSample.
+
+        **Examples:**
+
+        `[{"host": "localhost", "port": 24, "username": "me", "password": "secret"}]`
+
+        `[{"name": "myProducer", "host": "localhost", "port": 24, "username": "me", "password": "secret"}]`
+
+      </td>
+
+      <td>
+        \[]
+      </td>
+
+      <td style={{ "text-align": "center" }}>
+        M/I
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        **CONSUMERS**
+      </td>
+
+      <td>
+        Consumers to collect. For each consumer a `name`, `hostname`, `port`, `username`, and `password` can be specified in JSON form.
+        `name` is the consumer’s name as it appears in Kafka. If it is not set, metrics from all consumers in the host:port will be gathered.
+        `host`, `port`, `username`, and `password` are the optional JMX settings and use the default if unspecified.
+        It is also possible to set the value `default` to let the `name` undefined and use the default values for `host`, `port`, `username` and `password`.
+        Required to produce KafkaConsumerSample.
+
+        **Examples:**
+
+        `[{"host": "localhost", "port": 24, "username": "me", "password": "secret"}]`
+
+        `[{"name": "myConsumer", "host": "localhost", "port": 24, "username": "me", "password": "secret"}]`
+
+      </td>
+
+      <td>
+        \[]
+      </td>
+
+      <td style={{ "text-align": "center" }}>
+        M/I
+      </td>
+    </tr>
+
+    {
+      ' '
+    }
+
+    <tr>
+      <td>
+        **DEFAULT_JMX_HOST**
+      </td>
+
+      <td>
+        The default host to collect JMX metrics. If the host field is omitted from a
+        producer or consumer configuration, this value will be used.
+      </td>
+
+      <td>
+        localhost
+      </td>
+
+      <td style={{ 'text-align': 'center' }}>
+        M/I
+      </td>
+    </tr>
+
+    {
+      ' '
+    }
+
+    <tr>
+      <td>
+        **DEFAULT_JMX_PORT**
+      </td>
+
+      <td>
+        The default port to collect JMX metrics. If the port field is omitted from a
+        producer or consumer configuration, this value will be used.
+      </td>
+
+      <td>
+        9999
+      </td>
+
+      <td style={{ 'text-align': 'center' }}>
+        M/I
+      </td>
+    </tr>
+
+    {
+      ' '
+    }
+
+    <tr>
+      <td>
+        **DEFAULT_JMX_USER**
+      </td>
+
+      <td>
+        The default user that is connecting to the JMX host to collect metrics. If
+        the username field is omitted from a producer or consumer configuration,
+        this value will be used.
+      </td>
+
+      <td>
+        admin
+      </td>
+
+      <td style={{ 'text-align': 'center' }}>
+        M/I
+      </td>
+    </tr>
+
+    {
+      ' '
+    }
+
+    <tr>
+      <td>
+        **DEFAULT_JMX_PASSWORD**
+      </td>
+
+      <td>
+        The default password to connect to the JMX host. If the password field is
```

Comment:
If you omit the password field

Also, should the field names in this and the above ones be in code format?